docs/tools/piral-inspector.md

```diff
@@ -4,4 +4,215 @@ title: Piral Inspector
 
 # Piral Browser Extension
 
-(tbd)
+## Introduction
+
+To optimally work as a developer with your piral instance, and all of its containing pilets, Piral Inspector offers a great set of capabilities:
+- Getting information about the currently running Piral instance, e.g. name and version
+- Overview on currently loaded Pilets
+- Adding / Removing of pilets on-the-fly
+- Overview of registered routes
+- Collection of emitted events
+- Collection of global states
+
+As a developer of your piral instance, or your pilets, you are used to working with a web browser. And this is precisely the best place to get assisted by the Piral Inspector during the development or testing phase.
+
+We provide the Piral Inspector as an extension for the following browsers:
+- Google Chrome (>= v50)
+- Firefox (>= v50)
+- Opera (>= v50)
+
+
+## Installation of add-on / extension
```

Comment:
Title case: "Installation of Add-On / Extension"

Reply:
Done.